etl/steps/data/garden/tourism/2023-05-05/countries_regions.csv

```diff
@@ -0,0 +1,299 @@
+code,name,iso_alpha2,iso_alpha3,imf_code,cow_letter,cow_code,unctad_code,marc_code,ncd_code,kansas_code,penn_code,wikidata_uri,wikidata_label,legacy_entity_id,legacy_country_id,members,aliases
```

Comment:
This file should be deleted. You can still get them though! Just in a slightly different way. Try this instead
```
ds = Dataset(paths.LATEST_REGIONS_DATASET_PATH)
regions = ds["regions"]
regions.head()
```